### PR TITLE
add-and-remove-functions

### PR DESCRIPTION
**Pull request by mnaibei**
- Init project using webpack
- Created index and js files dynamically using webpack
- Created todo list
- User can add task
- User can remove task
- Tasks persist in local storage

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -32,6 +46,39 @@ const displayTasks = () => {
...
+    setTasksToStorage();
+    displayTasks();
+    inputField.value = '';
+  });
+};
+
+// clear complete tasks
+const clearTasks = () => {
+  tasks = tasks.filter((task) => task.completed === false);
+};
```
[the remark this answers is not in the document]
Comment (danmainah) [reply src/index.js:70]: - [ ] @mnaibei kindly add a button to delete an individual task currently to remove a task you need to mark it complete then click the `clear all completed` button

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi <STUDENT_MENTION>, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good design


# Required Changes ♻️

- [ ] @mnaibei great work on your documentation, however, your live demo link is broken, kindly fix it or remove the section since it's not a requirement for this milestone, also kindly specify your future feature javascript it's not clear.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by franclobo — CHANGES_REQUESTED** [review]
Hi @mnaibei, 

# ♻️ Required Changes ♻️

so-close-jim-halpert

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights

- No linter errors ✔️ 
- Descriptive PR ✔️ 
- Correct Github flow ✔️ 
- Good UI ✔️ 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/tasks.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  tasks.push(task);
+};
+
+// edit task description
+export const inlineEditTask = (tasks, id, newDesc) => {
+  const taskIndex = tasks.findIndex((task) => task.id === Number(id));
+  if (taskIndex !== -1) {
+    tasks[taskIndex].desc = newDesc;
+  }
+};
```
[the remark this answers is not in the document]
Comment (mnaibei) [reply src/modules/tasks.js:29]: Hi @franclobo thank you for your review., this part is giving me challenge I don't know why it returns undefined., can I work on fixing it later? or can you share some resources to help me fix it

**Comment by mnaibei** [comment]
**Hello Reviewer this is my final review, please let me know if you require any changes before you submit. 🙏 **

**Review by DuaneDave — APPROVED** [review]
Hi @mnaibei 👋 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ All requested changes have been addressed
✔️ App is working fine

## Before Delete

before

## After Delete

after



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**